raiden_contracts/contract_source_manager.py

```diff
@@ -184,3 +182,25 @@ def check_runtime_codesize(d: Dict) -> None:
         runtime_code_len = len(compilation["bin-runtime"]) // 2
         if runtime_code_len > 0x6000:
             raise RuntimeError(f"{name}'s runtime code is too big ({runtime_code_len} bytes).")
+
+
+def _verify_single_precompiled_checksum(
+    checked_checksums: Dict[str, str], contract_name: str, expected_checksum: str
+) -> None:
+    """ Get `checked_checksums[contract_name]` and compare it against `expected_checksum` """
+    try:
+        precompiled_checksum = checked_checksums[contract_name]
+    except KeyError:
+        raise ContractSourceManagerVerificationError(f"No checksum for {contract_name}")
+    if precompiled_checksum != expected_checksum:
+        raise ContractSourceManagerVerificationError(
+            f"checksum of {contract_name} does not match. got {precompiled_checksum} != "
+            "expected {expected_checksum}"
+        )
+
+
+def verify_single_precompiled_checksum_on_nonexistetnt_contract_name() -> None:
```

Comment:
It appears, this functions exists only for testing behavior. So it seems a little odd to have it inside `contract_source_manager.py` -- can't you move it to, e.g. https://github.com/raiden-network/raiden-contracts/blob/master/raiden_contracts/tests/utils/contracts.py ?

Reply:
I'm trying to avoid calling `_xyz` from other modules.  @hackaugusto what do you suggest?